app/components/certificate/request.tsx

```diff
@@ -0,0 +1,22 @@
+import { Flex, Heading, Text, HStack } from '@chakra-ui/react';
+
+interface CertificateRequestProps {
+  subject: String;
```

Comment:
`String` with a capital `S` is the JavaScript constructor function.  You want `string` lower-case 's' for TypeScript.